recipe/migrations/pypy38.yaml

```diff
@@ -0,0 +1,53 @@
+migrator_ts: 1647123563
+__migrator:
+    migration_number: 1
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy
+            - 3.8.* *_73_pypy
+            - 3.9.* *_73_pypy
+    paused: False
+    longterm: True
+    use_local: False
```

Comment:
@h-vetinari, change this to `True`

Reply:
Thanks!

Reply:
Seems to work: https://github.com/conda-forge/numpy-feedstock/pull/261